app/views/projects/_form_0.html.erb

```diff
@@ -228,7 +228,7 @@
                 # data is much more likely to be accurate.
                 f.text_field :license,
                              class: 'form-control', hide_label: true,
-                             pattern: '(|[()A-Za-z0-9\-.+ ]*)',
+                             pattern: '[()A-Za-z0-9\-.+ ]*',
```

Comment:
I think the trailing asterisk will cause us to match empty strings. Might be better off with `[()A-Za-z0-9\-.+ ]+` ?

Reply:
Matching empty strings is 100% intended.

Reply:
We need people to be able to save partially-completed forms.